config/routes.rb

```diff
@@ -10,6 +10,7 @@
   resources :feeds, only: %i(index)
   resources :users do
     resources :practices, only: %i(index), controller: 'users/practices'
+    resources :reports, only: %i(index), controller: "users/reports"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.